src/presentation/wallet/receive/lightning-enter-amount.tsx

```diff
@@ -0,0 +1,265 @@
+import Button from '../../components/button';
+import Boltz, { ReverseSubmarineSwapResponse } from '../../../application/utils/boltz';
+import { fromSatoshi, toSatoshi } from '../../utils/format';
+import { selectMainAccount } from '../../../application/redux/selectors/wallet.selector';
+import { useDispatch, useSelector } from 'react-redux';
+import { ProxyStoreDispatch } from '../../../application/redux/proxyStore';
+import { incrementAddressIndex } from '../../../application/redux/actions/wallet';
+import { broadcastTx } from '../../../application/utils/network';
+import { isSet, sleep } from '../../../application/utils/common';
+import LightningResultView from './lightning-result';
+import ModalUnlock from '../../components/modal-unlock';
+import { debounce } from 'lodash';
+import {
+  DEFAULT_LIGHTNING_LIMITS,
+  getClaimTransaction,
+  getInvoiceTag,
+  isValidReverseSubmarineSwap,
+} from '../../utils/boltz';
+
+export interface LightningAmountProps {
+  network: NetworkString;
+  explorerURL: string;
+}
+
+const LightningAmountView: React.FC<LightningAmountProps> = ({ explorerURL, network }) => {
+  const dispatch = useDispatch<ProxyStoreDispatch>();
+
+  const history = useHistory();
+  const account = useSelector(selectMainAccount);
+
+  const [errors, setErrors] = useState({ amount: '', submit: '' });
+  const [invoice, setInvoice] = useState('');
+  const [isModalUnlockOpen, showUnlockModal] = useState<boolean>(false);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [limits, setLimits] = useState(DEFAULT_LIGHTNING_LIMITS);
+  const [lookingForPayment, setIsLookingForPayment] = useState(false);
+  const [touched, setTouched] = useState(false);
+  const [txID, setTxID] = useState('');
+  const [values, setValues] = useState({ amount: '' });
+
+  const boltz = new Boltz(network);
+
+  useEffect(() => {
+    // get maximal and minimal amount for pair
+    const fetchData = async () => {
+      const pair = await boltz.getPair('L-BTC/BTC');
+      if (pair?.limits) {
+        setLimits({
+          maximal: fromSatoshi(pair.limits.maximal),
+          minimal: fromSatoshi(pair.limits.minimal),
+        });
+      }
+    };
+    fetchData().catch(console.error);
+  }, []);
+
+  const handleBackBtn = () => history.goBack();
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    event.preventDefault();
+
+    setTouched(true);
+
+    const value = event.target.value;
+
+    if (!isSet(value)) {
+      setTouched(false);
+      setErrors({ amount: '', submit: '' });
+      setValues({ amount: value });
+      return;
+    }
+
+    if (Number.isNaN(value)) {
+      setErrors({ amount: 'Not valid number', submit: '' });
+      setValues({ amount: value });
+      return;
+    }
+
+    if (Number(value) <= 0) {
+      setErrors({ amount: 'Number must be positive', submit: '' });
+      setValues({ amount: value });
+      return;
+    }
+
+    if (Number(value) <= limits.minimal) {
```

Comment:
```suggestion
    if (Number(value) < limits.minimal) {
```

I guess exactly the minimum is fine isn't it?

Reply:
Fixed by f33bd6e